### DISCUSSION TITLE
Svelte image processing like Next & Nuxt.

### DISCUSSION THREAD

**Discussion by HenrijsS** [sveltejs/kit#2994] [Ideas]
Next.js has an awesome [first-party image processing component](https://nextjs.org/docs/api-reference/next/image) that compresses and automatically resizes the images to fit the needs of the website.
Nuxt follows suite with [Nuxt/Image](https://image.nuxtjs.org/)

Both of these libraries seem to do the same thing.
Maybe there's a way to implement the [Sharp](https://sharp.pixelplumbing.com/) library and create a component with the same options? 
Is image optimization in-mind for Sveltekit overall?

**Comment by Theo-Steiner** [comment]
This is already being discussed in #241
 I think it would be best if you added your suggestion to that issue, so everything can be discussed in one place!

**Reply by HenrijsS** [reply]
Oh thanks.
That's weird.
I searched the discussions for `"image"` and it didn't return that discussion.

**Comment by mandrasch** [comment]
Btw, for anyone also finding this via search engines: This is also discussed here https://github.com/sveltejs/kit/discussions/10808 - next to https://github.com/sveltejs/kit/issues/241
